dbt/adapters/athena/impl.py

```diff
@@ -377,7 +377,7 @@ def _get_one_table_for_catalog(self, table: dict, database: str) -> list:
             "table_database": database,
             "table_schema": table["DatabaseName"],
             "table_name": table["Name"],
-            "table_type": self.relation_type_map[table["TableType"]].value,
+            "table_type": self.relation_type_map[table.get("TableType", "table")].value,
```

Comment:
```suggestion
            "table_type": self.relation_type_map[table.get("TableType", TableType.TABLE)].value,
```

Reply:
@svdimchenko in this case, I won't use the dict "value" to do the lookup, I know they match, but for example, originally I was thinking to use `EXTERNAL_TABLE` so for me it's a sort of circular loop - I'm using a dict value as a dict key :D 
How about using EXTERNAL_TABLE  instead as default value?

Reply:
ah, yeah, you are right. So let's keep your original "table" better

Reply:
I put EXTERNAL_TABLE, as if we think about it, it's an `EXTERNAL_TABLE`created from an external system.